src/interface/view/static/index.css

```
@@ -0,0 +1,446 @@
+@import url("https://fonts.googleapis.com/css2?family=PT+Mono&display=swap");
```

Stylelint (17.14.0)

[error] Expected "url("https://fonts.googleapis.com/css2?family=PT+Mono&display=swap")" to be ""https://fonts.googleapis.com/css2?family=PT+Mono&display=swap"" (import-notation)

(import-notation)